dapp_manager/exceptions.py

```diff
@@ -42,17 +45,27 @@ class StartupFailed(DappManagerException):
 
     SHELL_EXIT_CODE = 6
 
-    def __init__(self, stdout, stderr, runner_stdout, runner_stderr):
-        msg = (
-            "Dapp startup failed.\n"
-            "--- dapp-runner stderr ---\n"
-            f"{runner_stderr}\n\n"
-            "--- dapp-runner stdout ---\n"
-            f"{runner_stdout}\n\n"
-            "--- pre-runner stderr ---\n"
-            f"{stderr}\n\n"
-            "--- pre-runner stdout ---\n"
-            f"{runner_stdout}\n\n"
+    def __init__(self, stdout, stderr, runner_stdout: Optional[str], runner_stderr: Optional[str]):
+        if runner_stderr is None:
+            runner_stderr_text = "--- no dapp-runner stderr ---"
```

Comment:
in such case, I'd just have empty string here... so that we don't clutter the output... 

```suggestion
            runner_stderr_text = ""
```